lighthouse-cli/test/smokehouse/test-definitions/perf/expectations.js

```diff
@@ -170,4 +170,35 @@ module.exports = [
       },
     },
   },
+  {
+    lhr: {
+      requestedUrl: 'http://localhost:10200/perf/trace-nodes.html',
+      finalUrl: 'http://localhost:10200/perf/trace-nodes.html',
+      audits: {
+        'lcp-node': {
```

Comment:
shouldn't this be `largest-contentful-paint-element`? i guess you need to merge master?

Reply:
That was something I missed when I did the merge. Haven't fixed the smoke test yet so didn't realize it.